### ISSUE TITLE
Create functionality to record/facilitate periodic checking of MasterFiles on Stornext

### ISSUE THREAD

**Issue by acurley** [uvalib/tracksys#4]
We need to check for bit rot, so we should create a processor to periodically check for changes in md5.  Record the date last checked.

Also want to create a priority list for how important to check for this file.  Items in DL would have highest priorities, items without Bibl records that have catalog key have the lowest priority, etc....  Perhaps we should check 5 highest priority files a night, 4 of the next hightest, etc....  Make this processor contingent on running based on other Stornext work.  Add to same processor group?

**Comment by mikedurbin** [comment]
Stornext is no longer the storage system for this content.

This will be handled as needed with another effort lead by the preservation department.
